client-display/src/userSkills/skill/SelfReportSkillModal.vue

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2020 SkillTree
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+<template>
+  <b-modal id="reportSkillModal"
+           title="REPORT SKILL"
+           ok-title="Submit"
+           :no-close-on-backdrop="true"
+           v-model="modalVisible">
+    <div class="row p-2" data-cy="selfReportSkillMsg">
+      <div class="col-auto text-center">
+        <i v-if="isHonorSystem" class="fas fa-chess-knight text-success" style="font-size: 3rem"></i>
+        <i v-if="isApprovalRequired" class="fas fa-thumbs-up text-info" style="font-size: 3rem"></i>
+      </div>
+      <div class="col">
+        <p class="h5" v-if="isHonorSystem">This skill can be submitted under the <b class="text-success">Honor
+          System</b> and <b class="text-success">{{ skill.pointIncrement }}</b> points will apply right away!
```

Comment:
might be better phrased as "points will be awarded right away!"

Reply:
agreed!